helm/alfresco-content-services/7.3.N_values.yaml

```diff
@@ -76,6 +77,5 @@ postgresql-syncservice:
   image:
     tag: 14.4.0
 alfresco-sync-service:
-  syncservice:
-    image:
-      tag: 3.9.0
+  image:
+    tag: 3.8.0
```

Comment:
check this

Reply:
github glitch (probably due to some reverted commit that's been merged in master via another PR in the meantime)